etl/steps/data/meadow/oecd/2024-08-21/official_development_assistance.py

```diff
@@ -0,0 +1,182 @@
+"""Load a snapshot and create a meadow dataset."""
+
+from etl.helpers import PathFinder, create_dataset
+
+# Get paths and naming conventions for current step.
+paths = PathFinder(__file__)
+
+# Define the list of OECD datasets to upload, the file format, the columns to keep and the index columns.
+DATASETS = {
+    "dac1": {
+        "file_format": "zip",
+        "file_format_in_archive": "csv",
+        "file_name": "Table1_Data",
```

Comment:
not urgent, but you could simply define one property (e.g. `file_name`) which already combines the name and extension:

```yaml
file_name: "Table1_Data.csv"
```